elk/plotting/visualize.py

```diff
@@ -7,8 +7,8 @@
 import plotly.graph_objects as go
 from plotly.colors import qualitative
 from plotly.subplots import make_subplots
-
-from .utils import display_table
```

Comment:
Could you explain why you changed this? Do you prefer having less files rather than more?